packages/next/shared/lib/runtime-image-config-context.ts

```diff
@@ -0,0 +1,12 @@
+import React from 'react'
+import {
+  ImageConfigComplete,
+  imageConfigDefault,
+} from '../../server/image-config'
+
+export const RuntimeImageConfigContext =
```

Comment:
Lets call this `ImageConfigContext` so it can be used by both server and client

Reply:
Done, Rename to ImageConfigContext.